checkout-service/database/index.js

```diff
@@ -1,24 +1,26 @@
-const mongoose = require('mongoose');
-mongoose.connect('mongodb://localhost/fetcher');
-
-
-let checkoutSchema = mongoose.Schema({
-
+const Mongoose = require('mongoose');
+const Db = 'mongodb://localhost/prinventory';
+const mongooseOptions = {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+  useFindAndModify: false
+};
+
+Mongoose.connect(Db, mongooseOptions, (err) => {
+  if (err) {
+    console.error(err);
+  } else {
+    console.log('Connected to mongodb');
+  }
 });
 
-let Checkout = mongoose.model('Checkout', checkoutSchema);
-
 
+const PrinventorySchema = Mongoose.Schema({
+  id: Number,
+  price: Number,
+  inventory: Number,
+});
 
-// {
-//   ProductId: Number,
-//   Price: Number,
-//   Inventory: Number
```

Comment:
Please remove commented/unused code if possible

Reply:
This is from a previous commit